### PR TITLE
Unprotected read functionality

### PR DESCRIPTION
**Pull request by omarbowen01**

### CODE REVIEW

File: app/javascript/components/pages/EventShow.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+              <p>Event Name: {this.props.eventObj && this.props.eventObj.event_name}</p>
+                <NavLink to={`/Edit/${this.props.eventObj && this.props.eventObj.id}`}>
+                  <Button>Edit Event Listing</Button>
+                </NavLink>
+                <NavLink to={`/EventIndex/${this.props.eventObj && this.props.eventObj.id}`}>
+                  <Button onClick = {this.handleDelete} >
+                    Delete Event Listing
+                  </Button>
+                </NavLink>
+            </div>
```
Comment (WalkerTexas-coder) [inline app/javascript/components/pages/EventShow.js]: Indentation

File: app/javascript/components/pages/Show.js [hunk 2]
```diff
@@ -1,13 +1,49 @@
...
+                <NavLink to={`/Edit/${this.props.eventObj && this.props.eventObj.id}`}>
+                  <Button>Edit Event Listing</Button>
+                </NavLink>
+                <NavLink to={`/EventIndex/${this.props.eventObj && this.props.eventObj.id}`}>
+                  <Button onClick = {this.handleDelete} >
+                    Delete Event Listing
+                  </Button>
+                </NavLink>
+            </div>
+            </div>
```
Comment (WalkerTexas-coder) [inline app/javascript/components/pages/Show.js]: Indentation

**Review by WalkerTexas-coder — APPROVED** [review]
Clean up the code and you'll be good ot go!

File: app/javascript/components/App.js [hunk 3]
```diff
@@ -2,29 +2,76 @@ import React, { Component } from 'react'
...
+      }
+  }
+
+  componentDidMount(){
+    this.readEvent()
+  }
+
+  readEvent = () => {
+    this.setState({events: mockEvents})
+    // fetch("http://localhost:3000/events")
```
Comment (archae0pteryx) [inline app/javascript/components/App.js]: No unused code comments. It's bad form.

File: app/javascript/components/pages/Show.js [hunk 4]
```diff
@@ -1,13 +1,49 @@
...
+            about: "",
+            event_name: ""
+          },
+          submitted: false
+        }
+      }
+
+    render() {
+
+      console.log("propsforshowjs",this.props)
```
Comment (archae0pteryx) [inline app/javascript/components/pages/Show.js]: Putting a console.log in a method while you're deving is not optimal but it's okay. But putting a console log in the render is no bueno. You shouldn't commit console logs.

**Review by archae0pteryx — CHANGES_REQUESTED** [review]
There's a ton of formatting issues here. Like i mentioned in slack if you get a code formatter (like prettier) that will make all of this super easy. Or you can do it manually.

File: app/javascript/components/pages/EventIndex.js [hunk 5]
```diff
@@ -1,12 +1,53 @@
...
 
-  render() {
-    return(
-      <>
-        <h1>Hello this is the event index!</h1>
-      </>
+  handleSubmit = () =>  {
+    console.log("click")
+  }
+          render() {
```
Comment (archae0pteryx) [inline app/javascript/components/pages/EventIndex.js:10]: This indenting is still off. FYI.

**Review by archae0pteryx — APPROVED** [review]

**Pull request merged**